app/src/main/java/com/globant/scriptsapadea/ui/fragments/TakePictureFragment.java

```diff
@@ -0,0 +1,130 @@
+package com.globant.scriptsapadea.ui.fragments;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.os.Build;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.globant.scriptsapadea.R;
+import com.globant.scriptsapadea.manager.ActivityResultEvent;
+import com.globant.scriptsapadea.utils.ImageRealPath;
+import com.squareup.otto.Subscribe;
+
+/**
+ * Created by leonel.mendez on 5/19/2015.
+ */
+public class TakePictureFragment extends BaseFragment implements View.OnClickListener {
```

Comment:
@leomendezg views (like Button) should be the owners of his own click events not the hole screen.